dateutil/parser/_parser.py

```diff
@@ -1352,19 +1359,22 @@ def parse(self, tzstr):
                             setattr(res, offattr,
                                     (int(l[i]) * 3600 +
                                      int(l[i + 2]) * 60) * signal)
+                            used_idxs.append(i)
                             i += 2
                         elif len_li <= 2:
                             # -[0]3
                             setattr(res, offattr,
                                     int(l[i][:2]) * 3600 * signal)
                         else:
                             return None
+                        used_idxs.append(i)
                         i += 1
                     if res.dstabbr:
                         break
                 else:
                     break
 
+
             if i < len_l:
                 for j in range(i, len_l):
                     if l[j] == ';':
```

Comment:
Can I eliminate this as a possibility for an alternate separator? Is not being hit by the test and clearly does not work if you consistently use ";" as a separator.

Reply:
I really wish I knew, honestly. :(